src/main/scala/fm/http/MimeTypes.scala

```diff
@@ -82,6 +83,11 @@ object MimeTypes {
     DOCX        -> "docx"
   ).toUniqueHashMap
 
+  // type -> charset
+  val mimeTypeToCharset = Vector[(String, Charset)](
+    JSON -> StandardCharsets.UTF_8 // http://www.ietf.org/rfc/rfc4627.txt "JSON text SHALL be encoded in Unicode.  The default encoding is UTF-8."
+  ).toUniqueHashMap
```

Comment:
It seems like if we flipped generateSOAP back to true in TecDocAdminWS, then the XML response would be written with the default ISO-1859-1 charset set in this library, and the French characters (and others) would not render correctly. Is there a way for a client of this library to override this default?

Reply:
I think you're missing something.  The behavior is suppose to match the HTTP specification here.  Specifically for TecDocAdminWS, if you also look at their HTTP response:

```
22:01:37.952 [scala-execution-context-global-145] DEBUG ta.webservice.admin.v1.TecDocAdminWSBareSOAPClient - Response Headers: Date: Tue, 17 Apr 2018 22:01:37 GMT
Server: Apache
Content-Length: 0
Connection: close
Content-Type: text/plain; charset=UTF-8
```